assignments/callbacks.js

```diff
@@ -39,31 +39,71 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-}
+// function getLength(items, callback) {
+//   return callback(items);
+// }
+// const callback = (items) => {
```

Comment:
The reason why the code throws an error when this bunch is uncommented is because there are two of these:
```
const callback = 
```
JS only lets you declare a constant or variable once, so the easy fix for this is to just change the name of the function to something like `callback2`. That way you won't have two variables with the same name 👍